packages/viewer/src/components/viewer-markup-circle/viewer-markup-circle.tsx

```diff
@@ -95,11 +96,11 @@ export class ViewerMarkupCircle {
   public editEnd!: EventEmitter<void>;
 
   /**
-   * An event that is dispatched when the user cancels editing of the
-   * markup.
+   * An event that is dispatched when this markup element is in view
+   * mode (`this.mode === ""`), and it completes a rerender.
    */
   @Event({ bubbles: true })
-  public editCancel!: EventEmitter<void>;
+  public viewRendered!: EventEmitter<void>;
```

Comment:
How will this be used?

Reply:
This is being used to know when the newly created markup has been appended to the `<vertex-viewer-markup>` element and completed a render, so we can reset the "draft" markup in the `<vertex-viewer-markup-tool>`. This fixes the "flicker" you see when creating a markup currently